plugins/prefetch/plugin.cc

```diff
@@ -450,7 +499,7 @@ contHandleFetch(const TSCont contp, TSEvent event, void *edata)
     }
     if (!appendCacheKey(txnp, reqBuffer, data->_cachekey)) {
       PrefetchError("failed to get the cache key");
-      TSHttpTxnReenable(txnp, TS_EVENT_HTTP_ERROR);
+      TSHttpTxnReenable(txnp, TS_EVENT_HTTP_CONTINUE);
```

Comment:
This might be a surprising change to existing users.  Can you explain why this is necessary?

Reply:
A failed prefetch was causing the original transaction to abort.  I'm sure that's undesireable behavior.

Reply:
i just restored this.